web3swift/src/Client/JSONRPC.swift

```diff
@@ -25,9 +25,21 @@ public struct JSONRPCResult<T: Decodable>: Decodable {
     var result: T
 }
 
-public struct JSONRPCErrorDetail: Decodable {
+public struct JSONRPCErrorDetail: Decodable, Equatable, CustomStringConvertible {
     var code: Int
```

Comment:
`code` and `message` should be public as well to provide more flexibility to consumers since we're now exposing it.

Reply:
👍